hue/templates/configmap.yaml

```diff
@@ -191,15 +138,15 @@ data:
       livy_server_session_kind={{ .Values.config.spark.livyServerSessionKind }}
 
       # Whether Livy requires client to perform Kerberos authentication.
-      security_enabled={{ .Values.config.spark.securityEnabled }}
+      #security_enabled={{ .Values.config.spark.securityEnabled }}
 
       # Host of the Sql Server
-      sql_server_host={{ .Values.config.spark.sqlServerHost }}
+      #sql_server_host={{ .Values.config.spark.sqlServerHost }}
```

Comment:
if you want to remove this, you should do it via a conditional

Reply:
We don't really want this configuration at the moment as it is related with hive. I'll better remove them.

Reply:
BTW this is one of the configuration that even leaving it to blank causes GUI errors.